gene_benchmark/tasks.py

```diff
@@ -88,6 +88,19 @@ def _convert_df_to_3d_array(df: pd.DataFrame):
     return np.hstack(reshaped_embeddings)
 
 
+def _get_none_nan_instancies(
+    encodings: pd.DataFrame, outcomes: pd.Series | pd.DataFrame
+):
+    if encodings.shape[1] == 1:
+        nan_ind = encodings.squeeze().isna()
```

Comment:
its hard to understand this logic, encodings.shape[1] == 1 , why does this need squeeze? 
cant we do this: 
```
# Find indices of rows with any NaN values
nan_rows = df.isna().any(axis=1)

# Get the indices of these rows
nan_indices = df.index[nan_rows]
```

Reply:
`nan_rows = df.isna().any(axis=1)`
 solves it really nicely thanks for the syntax